assets/src/dashboard/app/views/exploreTemplates/index.js

```diff
@@ -61,6 +64,15 @@ function ExploreTemplates() {
     });
   }, [templatesOrderById, templates]);
 
+  const handlePreviewTemplate = useCallback(
+    (template) => {
+      if (enableTemplatePreviews) {
+        preview.set(template);
+      }
+    },
+    [enableTemplatePreviews, preview]
+  );
```

Comment:
I'm wondering if it's a little safer to do this flag check inside `preview.set()` . Probably gucci for now, but could be nice to set the flag branching behavior a little closer to the root of the functionality if we start using `preview.set()` in a bunch of other places

Reply:
I agree with this.  I think the cleanest way to use the feature flags is to disable the feature at its core so that if any code even tries to trigger it, it just won't work, rather than having to wrap each triggering point with an if statement (that could easily be forgotten).  

Like with Max, I agree this doesn't need to be done, but in general, disabling the `preview.set` functionality feels like the cleanest way to prevent previews from showing up no matter which button actually triggers the preview.

Reply:
great idea! thanks guys!